docs/user/reference/general/initializing.md

```diff
@@ -174,8 +178,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 ```
 
-The Glean Swift SDK does not support use across multiple processes,
-and must only be initialized on the application's main process.
+The Glean Swift SDK supports use across multiple processes. To configure this you call `initialize` as usual from your main process. From the secondary process you can call `initialize` again with the `dataPath` parameter. This allows you to store data locally in a separate location for the secondary process.
+
+Requirements for a secondary process
+- `initialize` must be called from the main thread
```

Comment:
Does it _really_ need to be called on the main thread on a background process?

Reply:
This is listed in Jan-Erik's proposal as a requirement, so I am just going based off of that.

@badboy 
Is there any specific reason it needs to be called from the main thread?

Reply:
Looking at it now and I can't really say why I made it a requirement.
I think it's fine in NOT being on the main thread. We can remove the check here and adjust the proposal retroactively.

In the worst case we need to change it later, but we know the users needing this so we can help them in that case.

Reply:
Fixed in https://github.com/mozilla/glean/pull/2433/commits/90da5a9474a9fd02651560275dc902911c04e2e8